### PR TITLE
Executable on older Windows

### PR DESCRIPTION
**Pull request by michal367**
Older Windows e.g. Windows 7 needs api-ms-win-* and ucrtbase dlls that were missing.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # validation page for appveyor config: https://ci.appveyor.com/tools/validate-yaml
 
+image: Visual Studio 2019
```
Comment (grzanka) [inline appveyor.yml:3]: please explain in comment why this particular VS is needed

File: appveyor.yml [hunk 2]
```diff
@@ -56,7 +58,7 @@ build_script:
 # installer
 # generate single dir installation
 # add InnoSetup to PATH
-  - set PATH="C:\Program Files (x86)\Inno Setup 5";%PATH%
+  - set PATH="C:\Program Files (x86)\Inno Setup 6";%PATH%star
```
Comment (grzanka) [inline appveyor.yml:61]: do I see correctly `star` appended ?

**Review by grzanka — CHANGES_REQUESTED** [review]

File: appveyor.yml (hunk 2 above)
Comment (rozchlastywacz) [reply appveyor.yml]: So do I :)

**Review by grzanka — APPROVED** [review]
ok

**Pull request merged**